features/nationaliteithistorie/dev/redenBeeindigen.feature

```diff
@@ -16,17 +16,21 @@ Functionaliteit: Lever de juiste gegevens over een nationaliteit
   Rule: Voor een beëindigde nationaliteit of beëindigd bijzonder Nederlanderschap wordt de redenBeeindigen overgenomen uit de actuele categorie (04).
 
     Scenario: persoon heeft een beëindigde nationaliteit met gecorrigeerde reden beëindigen
-      Gegeven de persoon met burgerservicenummer '999994657' heeft een 'nationaliteit' met de volgende gegevens
+      Gegeven de persoon met burgerservicenummer '000000139' heeft een 'nationaliteit' met de volgende gegevens
+      | nationaliteit (05.10) | reden opnemen (63.10) | datum ingang geldigheid (85.10) |
+      | 0100                  | 301                   | 19890301                        |
+      En de 'nationaliteit' is gewijzigd met de volgende gegevens
       | nationaliteit (05.10) | reden opnemen (63.10) | reden beëindigen (64.10) | datum ingang geldigheid (85.10) |
-      |                       |                       | 404                      | 20140601                        |
-      |                       |                       | 401                      | 19940601                        |
-      | 0100                  | 301                   |                          | 19890301                        |
+      |                       |                       | 401                      | 20220601                        |
+      En de 'nationaliteit' is vervolgens gewijzigd met de volgende gegevens
+      | reden beëindigen (64.10) | datum ingang geldigheid (85.10) |
+      | 404                      | 20220601                        |
       Als nationaliteithistorie wordt geraadpleegd met de volgende parameters
-      | naam                | waarde                                                                  |
-      | type                | RaadpleegMetPeriode                                                     |
-      | burgerservicenummer | 999994657                                                               |
-      | peildatum           | 2000-01-01                                                              |
-      | fields              | nationaliteiten.nationaliteit.code,nationaliteiten.redenBeeindigen.code |
+      | naam                | waarde                               |
+      | type                | RaadpleegMetPeriode                  |
+      | burgerservicenummer | 000000139                            |
+      | peildatum           | 2000-01-01                           |
+      | fields              | nationaliteiten.redenBeeindigen.code |
       Dan heeft de response de volgende 'nationaliteiten'
-      | type          | nationaliteit.code | redenBeeindigen.code |
-      | Nationaliteit | 0100               | 404                  |
+      | type          | redenBeeindigen.code |
+      | Nationaliteit | 404                  |
```

Comment:
Hier moet volgens mij de peildatum of het resultaat aangepast worden. Als de peildatum na 01-06-2022 ligt klopt het resultaat.

Reply:
na 01-06-2022 is er volgens dit testgeval helemaal geen nationaliteit meer, dus zou je die niet krijgen.

vraag hier is hoe we historie weergeven
1. alleen gegevens tonen die op peildatum of in periode geldig waren, 
2. peildatum/periode bepaalt alleen welke nationaliteit je te zien krijgt, elke nationaliteit geleverd inclusief alles wat daarna aan die nationaliteit is gewijzigd.

Dat laatste is hoe het nu beschreven staat en ook in v1.3 in de features stond.

| situatie | leveren bij 1 | leveren bij 2 |
|-- |-- |-- |
| peildatum ligt voor beëindigen | nationaliteit zonder redenBeeindigen en datumTot | nationaliteit inclusief redenBeeindigen en datumTot |
| peildatum ligt na beëindigen | geen nationaliteit | geen nationaliteit |
| periode ligt voor beëindigen | nationaliteit zonder redenBeeindigen en datumTot | nationaliteit inclusief redenBeeindigen en datumTot |
| periode ligt na beëindigen | geen nationaliteit | geen nationaliteit |
| einddatum ligt binnen periode | nationaliteit inclusief redenBeeindigen en datumTot | nationaliteit inclusief redenBeeindigen en datumTot |